WcaOnRails/app/models/delegate_report.rb

```diff
@@ -25,6 +25,59 @@ def equipment_default
 Gen 3 Display: 0"
   end
 
+  before_create :venue_default
+  def venue_default
+    self.venue = "* **Previous Competitions in the same venue?:**
```

Comment:
Would it make sense to have markdown partials for placeholders instead of hardcoding such long strings?
It would make it easy and clear what needs editing to change the information in the placeholder.

I managed to get it working locally by adding a `_venue_default.md` file under `app/views/delegate_reports`, and using the following code:
```ruby
  before_create :venue_default
  def venue_default
    self.venue = ActionController::Base.new().render_to_string(template: "delegate_reports/_venue_default.md")
  end
```

Kind of breaking MVC, but it would look much better in my opinion (and would not run into the same linelength issue I think, since it would be in markdown files).

Reply:
@jfly any opinion on this?

Reply:
Sounds good to me, definitely a better alternative to keeping all the text in the model! No preference whether it should be under views or just a file somewhere close to the model itself though.

Reply:
I like this as well!